easybuild/tools/module_generator.py

```diff
@@ -1206,7 +1227,8 @@ class ModuleGeneratorLua(ModuleGenerator):
     LOAD_TEMPLATE_DEPENDS_ON = 'depends_on("%(mod_name)s")'
     IS_LOADED_TEMPLATE = 'isloaded("%s")'
 
-    OS_GETENV_TEMPLATE = r'os.getenv("%s")'
```

Comment:
Same here, this is part of the API, we shouldn't just remove this...

Reply:
done in https://github.com/easybuilders/easybuild-framework/pull/5049/commits/67c8994a6f60b16a57c04b262edfadea3e2ed931